src/background/services/overlayService.ts

```diff
@@ -91,6 +91,18 @@ export class OverlayService {
     return this._openOverlay('OPEN_PAIRING_OVERLAY', { topic, args })
   }
 
+  public openConnectedAccountsPopup(
+    { accountsToConnect, accountsToDisconnect, accountToChangeStatus },
+    tabId: number
+  ) {
+    return this._openOverlay(
+      'OPEN_SYSTEM_OVERLAY',
```

Comment:
I think we need to eliminate the legacy `iframe`-based overlay in the future and move all popups to new React-based overlay written by Galina. What do you think?

Reply:
https://dapplets-org.atlassian.net/browse/DAP-2191
😉